moveToWebcompat.js

```diff
@@ -86,42 +111,89 @@ const sendToWebcompat = async bug => {
     steps_reproduce: steps
   };
 
-  const form = convertToFormData(data);
-
   return await fetch(WEBCOMPAT_ENDPOINT, {
     method: "POST",
-    body: form
+    body: convertToFormData(data)
   });
 };
 
 const onMoveButtonClick = async () => {
   const bugId = getBugId();
-  if (!bugId)
-    openDropDown(
+  if (!bugId) {
+    return showError(
       "A valid bug ID could not be found. Please check you're actually viewing a bug."
     );
+  }
 
-  const result = await getBugData(bugId);
+  startLoadingState();
```

Comment:
This function name makes it sound a little more scary than what it really is doing; maybe we should change the name of this and the stop function to something like "disableMoveButton" and "enableMoveButton"?

Reply:
Yeah good point, thanks :)